core/shared/src/main/scala/cats/parse/Parser.scala

```diff
@@ -1446,12 +1456,23 @@ object Parser {
     *  this is useful for parsing comment strings, for instance.
     */
   def until0(p: Parser0[Any]): Parser0[String] =
-    (not(p).with1 ~ anyChar).rep0.string
+    (not(p).void.with1 ~ anyChar).rep0.string
```

Comment:
you don't need this BTW, the `.string` would internally call .void.

Reply:
good point!